src/script/script.cpp

```diff
@@ -592,6 +624,27 @@ bool AmountFromReissueScript(const CScript& scriptPubKey, CAmount& nAmount)
     nAmount = asset.nAmount;
     return true;
 }
+
+int SearchForRVN(const CScript& script, const int startingValue) {
+
+    // Initialize the start value
+    int index = -1;
```

Comment:
Might be overkill, but would it be worth it to do a bounds check on `startingvalue` to make sure it doesn't exceed the size of `script` before the first check?

Reply:
Agree -- will definitely not hurt.